static/docs/command-reference/update.md

```diff
@@ -39,6 +39,12 @@ to learn how to "update" fixed-revision imports.
 
 ## Options
 
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
+  (such as a branch name, a tag, or a commit hash) of the DVC repository to
+  update the data from. Repository's revision formed by the first import is used
```

Comment:
do we lock a specific revision for `import-url` or for `dvc import something` (no `--rev`)? It's not 100% clear.

Reply:
I created a discussion for choosing a behavior here: https://github.com/iterative/dvc/issues/2849#issuecomment-570808153

Reply:
changed by 82c8b5c, can you please re-check?